style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.profile {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 45px;
+  font-family: "poppins", sans-serif;
+  line-height: 50px;
+}
+
+.dev {
+  font-family: "poppins", sans-serif;
+  margin-top: 12px;
+  font-size: 16px;
+  line-height: 23px;
+  color: #344563;
+}
+
+.connect {
+  margin-top: 15px;
+  color: #7f8cff;
+}
+
+main {
+  background-image: url(./images/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+}
+
+section ul {
+  display: flex;
+  gap: 20px;
+  margin-top: 12px;
+}
+
+.pix {
+  height: 10px;
+}
+
+section {
+  margin: 128px 24px 136px 24px;
+}
+
+header a {
+  margin-top: 58px;
+  margin-left: 24px;
+  font-family: "poppins", sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+}
```

Comment:
- [ ] Kindly make sure to follow the exact styling as the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
You should follow the font's typography, size, weight, and margins, and paddings.
Please apply the corresponding changes on the logo as the given template.
### **Hint**
Pay attention to the design's paddings and widths to deliver the same results. Currently, the background image doesn't fully appear as it is in the design template. Am I right?
### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/120701446-21314800-c4b3-11eb-833d-2a84afd16b4b.png)
### **Original design**
![Screenshot](https://user-images.githubusercontent.com/47625407/120701473-2b534680-c4b3-11eb-855b-caed4b00d84b.png)